model/src/pyrenew/deterministic/deterministicpmf.py

```diff
@@ -18,6 +18,8 @@ def __init__(
         vars: ArrayLike,
         name: str,
         tol: float = 1e-5,
```

Comment:
Out of scope here but this should presumably have a default set higher up. See also #268